### PR TITLE
CHEMH-42 | @jdwjdwjdw | Update search and search-results page

### PR DESCRIPTION
**Pull request by jdwjdwjdw**
# READY FOR REVIEW

# Summary
- [CHEMH-13](https://stanfordits.atlassian.net/browse/CHEMH-13): Search restyling
- [CHEMH-42](https://stanfordits.atlassian.net/browse/CHEMH-42): Search results page

# Review By (Date)
- When convenient

# Review Tasks
## Setup tasks and/or behavior to test

1. Check out this branch
2. Rebuild Cache and import config `drush cr ; drush ci`
3. Review the search dropdown in the main menu, and confirm it matches the designs and works as intended
4. Review the search dropdown in the mobile menu, and confirm it matches the designs and works as intended
5. Review the search results page, and confirm it matches the designs and works as intended
6. Review code 🤖 

# Associated Issues and/or People
- [CHEMH-13](https://stanfordits.atlassian.net/browse/CHEMH-13): Search restyling
- [CHEMH-42](https://stanfordits.atlassian.net/browse/CHEMH-42): Search results page

[CHEMH-42]: https://stanfordits.atlassian.net/browse/CHEMH-42?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
[CHEMH-42]: https://stanfordits.atlassian.net/browse/CHEMH-42?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

[CHEMH-13]: https://stanfordits.atlassian.net/browse/CHEMH-13?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Comment by jenbreese** [comment]
I noticed a couple of things that seemed pretty off. I did switch back to v 2.x to verify it wasn't there. 
1. The search at a med breakpoint seems kind of big. 
2. There are search artifacts in the secondary menus.
3. The top nav has some odd indenting for level 2 menus. 
search-med-breakpoint
search-close-btn-side-nav
Seach-btn-side-nav
Drop-down-weird

File: chem_h_subtheme.theme [hunk 1]
```diff
@@ -79,6 +79,10 @@ function chem_h_subtheme_theme_suggestions_block_alter(array &$suggestions, arra
   }
 }
 
+function chem_h_subtheme_theme_suggestions_form_alter(array &$suggestions, array $variables) {
+  //dpm($variables);
```
Comment (jenbreese) [inline chem_h_subtheme.theme]: This comment should be removed.

File: src/scss/components/search/_search.scss [hunk 2]
```diff
@@ -1,67 +1,129 @@
...
 }
+
+// Mobile menu search
+.su-mobile-site-search {
+  #block-stanford-basic-search-mobile {
+    width: auto;
+    margin-top: 0;
+    margin-bottom: 0;
+  }
+}
```
Comment (jenbreese) [inline src/scss/components/search/_search.scss]: We need to have a blank line at the end of our SCSS file.

**Review by jenbreese — COMMENTED** [review]
I found a couple odd things in the display and a couple of code comments that I added at the line level.

**Comment by jenbreese** [comment]
I found a couple of things on the search results page.
1. The query is no longer in the search box. Here is how it looks on the live site.
2. The font sizes do not match the design files.
3. The vertical spacing between each entry is bigger than the design file. 
4. The search entry blurb doesn't stretch across the page to under the Last Updated.
Screen Shot 2023-04-19 at 10 03 19 AM
Screen Shot 2023-04-19 at 10 05 02 AM

**Comment by jdwjdwjdw** [comment]
Thanks @jenbreese! I think I addressed all the issues you mentioned, as well as a few other things we noticed like the icon size. Should be good to re-review now.

**Review by jenbreese — APPROVED** [review]
GtG.  Looks great!

**Pull request merged**